src/plot_api/helpers.js

```diff
@@ -0,0 +1,401 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var m4FromQuat = require('gl-mat4/fromQuat');
+
+var Registry = require('../registry');
+var Lib = require('../lib');
+var Plots = require('../plots/plots');
+var Axes = require('../plots/cartesian/axes');
+var Color = require('../components/color');
+
+
+// Get the container div: we store all variables for this plot as
+// properties of this div
+// some callers send this in by DOM element, others by id (string)
+exports.getGraphDiv = function(gd) {
```

Comment:
Moving these out of `plot_api.js` wasn't necessary, but it did made browsing over `plot_api.js` a little less painful.

Down the road (i.e. after https://github.com/plotly/plotly.js/milestone/9 is done), I think we should move each `Plotly` method to its own file in `src/plot_api/`.